README.md

```diff
@@ -136,6 +139,61 @@ func main() {
 	// Use client...
 }
 ```
+*Note*: In order to interact with certain apis, for example writing a file to a repo. One must generate an installation token
+using the installation id of the github app and authenticate with the OAuth method mentioned above. See the examples
+for more details.
+
+```go
```

Comment:
Please delete this entire section since the added example app covers these details.